### PR TITLE
Update Hybrid Kaleido ADM for DRE

### PR DESCRIPTION
**Pull request by dmjoy**
Updated Hybrid Kaleido ADM to use the Outlines ADM for action parameter completion, and update to make use of KDE based alignment utils for SoarTech scenarios. (WIP)

### CODE REVIEW

**Comment by dmjoy** [comment]
This PR is still a WIP, but not expecting any more changes to the outlines ADM.  Wanted to give you all (@eveenhuis @jadie1 @barry-ravichandran ) a chance to see specifically the outlines ADM changes to make sure your ADMs will not be affected.  My intent here was just to move the action parameter population pieces into their own methods so that I could re-use them in the Hybrid Kaleido ADM

File: align_system/algorithms/kaleido_adm.py [hunk 1]
```diff
@@ -276,25 +309,75 @@ def choose_action(self, scenario_state, available_actions, alignment_target, **k
             decision_environment=scenario_state.environment.decision_environment.unstructured.strip(),
             characters_str=characters_str)
 
-        # Scaling KDMA values by 10 (range should be 0-10)
-        if not isinstance(alignment_target, dict):
-            alignment_target = alignment_target.to_dict()
+        choices_unstructured = [a.unstructured for a in available_actions]
```
Comment (eveenhuis) [inline align_system/algorithms/kaleido_adm.py:312]: Do we need to ensure these choices are unique like we do for Outlines?

**Review by eveenhuis — COMMENTED** [review]
LGTM other than my one minor question

File: align_system/algorithms/kaleido_adm.py (hunk 1 above)
Comment (dmjoy) [reply align_system/algorithms/kaleido_adm.py:312]: Ah good catch, yes I think that makes sense.  Looks like just using the outlines_adm `format_choices` function should cover it yes?

Comment (eveenhuis) [reply align_system/algorithms/kaleido_adm.py:312]: I think so

**Pull request merged**